robocrys/condense/component.py

```diff
@@ -183,7 +183,10 @@ def get_sym_inequiv_components(
     components = deepcopy(components)
 
     sym_inequiv_components = {}
-    equivalent_atoms = spg_analyzer.get_symmetry_dataset()["equivalent_atoms"]
+    if hasattr(symm_dset := spg_analyzer.get_symmetry_dataset(),"equivalent_atoms"):
```

Comment:
alternatively, you could pin `spglib>=2.5` in `setup.py` and only support the attr version

Reply:
No particular preference from my side, maybe it's better to be more flexible wrt spglib version? Happy to go with your rec

Reply:
I would also lean towards Janosh's rec on this on one